tests/tracking/test_tracking.py

```diff
@@ -590,6 +590,114 @@ def test_log_figure_raises_error_for_unsupported_figure_object_type():
         mlflow.log_figure("not_figure", "figure.png")
 
 
+@pytest.mark.large
+@pytest.mark.parametrize("subdir", [None, ".", "dir", "dir1/dir2", "dir/.."])
```

Comment:
Could we also add tests for some of the `if` branches in the `log_image` implementation, to ensure we have thorough [branch coverage](https://en.wikipedia.org/wiki/Code_coverage#Basic_coverage_criteria)? For example, can we add a test for the clipping behavior?

Reply:
Actually, I think ^the other branches are covered pretty well, besides the clipping behavior